tools/pagerduty-migrator/migrator/report.py

```diff
@@ -47,9 +47,13 @@ def format_escalation_policy(policy: dict) -> str:
 
 
 def format_integration(integration: dict) -> str:
-    result = integration["service"]["name"] + " - " + integration["name"]
+    result = integration["service"]["name"] + " " + str(integration["name"])
 
-    if not integration["oncall_type"]:
+    if not "oncall_type" in integration.keys():
+        result = "{} {} — integration type not found. Manual webhook inserted.".format(
```

Comment:
Not really sure about falling back to the "webhook" integration type, probably we could make this as a feature flag in an env variable? Something like FALLBACK_TO_WEBHOOK_INTEGRATION_TYPE and make this as a separate PR? The idea behind the integration type matching inititally was to match them 1 to 1 without any guessing or fallbacks.